src/main/scala/chisel3/package.scala

```diff
@@ -430,8 +430,8 @@ package object chisel3 {    // scalastyle:ignore package.object.name
     val dontTouch = chisel3.core.dontTouch
 
     type BaseModule = chisel3.core.BaseModule
-    type MultiIOModule = chisel3.core.ImplicitModule
-    type RawModule = chisel3.core.UserModule
+    type MultiIOModule = chisel3.core.MultiIOModule
+    type RawModule = chisel3.core.RawModule
```

Comment:
This is the biggest of nits, but I used to confuse myself everytime I looked at this because these aren't sorted topologically by inheritance. Consider putting `RawModule` before `MultiIOModule`.